hummingbird/forms.py

```diff
@@ -0,0 +1,25 @@
+from django import forms
```

Comment:
nit summary:
- Reorder imports
- Two newlines between classes and between imports and first class
- One newline after class name
- End the file with a newline